src/App.js

```diff
@@ -1,42 +1,63 @@
+import {useEffect, useState} from 'react';
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
-import TodoListItem from './todoListItem';
+import TodoListItem from './TodoListItem';
```

Comment:
Without renaming the todoListItem.js file to TodoListItem.js this will break the app when trying to run it locally.